integration/Makefile

```diff
@@ -22,48 +22,48 @@ ci-integration-test: access-tests ghost-tests mvp-tests epochs-tests consensus-t
 # Run unit tests for test utilities in this module
 .PHONY: test
 test:
-	go test $(if $(VERBOSE),-v,) -tags relic -coverprofile=$(COVER_PROFILE) $(RACE_FLAG) $(if $(JSON_OUTPUT),-json,) $(if $(NUM_RUNS),-count $(NUM_RUNS),) `go list ./... | grep -v -e integration/tests`
```

Comment:
why enable this by default? wont this get very noisy in the CI logs?

Reply:
The VERBOSE flag was never set in CI so it was never verbose. We have had CI test failures without verbose logging that made it difficult to determine what was wrong. This shouldn't add a lot of extra logging, just the name of each test as it's being executed and a more explicit failure message. See https://github.com/onflow/flow-go/pull/3585#discussion_r1037417458